gr-blocks/lib/throttle_impl.cc

```diff
@@ -44,30 +55,44 @@ throttle_impl::~throttle_impl() {}
 bool throttle_impl::start()
 {
     d_start = std::chrono::steady_clock::now();
-    d_total_samples = 0;
```

Comment:
@marcusmueller The removal of this line has broken pause/resume when playing back I/Q files in Gqrx: https://github.com/gqrx-sdr/gqrx/issues/1234

I believe it should have been replaced with `d_total_items = 0`.